### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 ![Files](Files/Assets/FilesHome.png)
 
 ## What is Files?
-Files is a file manager which leverages the latest features of the Windows platform including Fluent Design, seamless updates, and APIs which enable the performance and lifecycle behavior that users expect. Whether you want to simplify your experience with your files or try something new, Files is a one-stop solution for exploring your files on the fly.
+Files is a file manager with a powerful yet intuitive design. It has features like multiple tabs, panes, columns, shell extensions in the context menu and tags.
```
Comment (d2dyno1) [inline README.md:13]: Packed with features like...

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Our vision for Files is to be fully functional while keeping it simple and easy
 ## Why should I contribute?
 We can confidently assert Files is the platform's best file explorer project to bring your innovations to. We focus heavily on extensibility and code modularity and have no plans to stop doing so. When you contribute to this project, you're helping everyone by fixing reported bugs, adding new features, or correcting existing behavior. These changes are quickly included in the final product for all users to benefit from.
 
-We welcome discussions and contributions on our repository, however to help maintain a healthy community, please check out our [code of conduct](https://github.com/files-community/Files/blob/main/CODE_OF_CONDUCT.md).
+We welcome discussions and contributions on our repository, however to help maintain a healthy community, please read our [code of conduct](https://github.com/files-community/Files/blob/main/CODE_OF_CONDUCT.md).
```
Comment (d2dyno1) [inline README.md:24]: We welcome discussions and contributions **on** -> **to** our repository | If I'm correct?

File: README.md (hunk 1 above)
Comment (d2dyno1) [reply README.md:13]: I'm not so sure about the above features and how it is spelled

Comment (d2dyno1) [reply README.md:13]: Also, I liked the last sentence of the old text, could you squish it somewhere?

> Whether you want to simplify your experience with your files or try something new, Files is a one-stop solution for exploring your files on the fly.

Comment (yaichenbaum) [reply README.md:13]: The text has already been decided and we've been using it in the store page, this PR is just to match the changes.

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**